grpc-sys/src/lib.rs

```diff
@@ -16,7 +16,7 @@
 #![allow(non_upper_case_globals)]
 #[allow(clippy::all)]
 mod bindings {
-    include!(concat!(env!("OUT_DIR"), "/grpc-bindings.rs"));
+    include!(concat!(env!("BINDING_DIR"), "/grpc-bindings.rs"));
```

Comment:
Better give it a platform-related name. For example, how to name the file if we want to also support macos?

Reply:
All Done, PTAL @BusyJay